backend/src/models/userModel.ts

```diff
@@ -20,8 +21,9 @@ const userSchema: Schema = new mongoose.Schema({
   email: { type: String, required: true, unique: true },
   password: { type: String, required: true },
   birthday: { type: Date, required: true },
-  friends: [{ type: String }], // Array of strings for storing user IDs
-  friendRequests: [{ type: String }], // Array of strings for storing user IDs
+  friends: [{ type: String }],
+  friendRequests: [{ type: String }],
```

Comment:
**Question: ❓ ** is friendRequests the requests that the user has received and pendingFriendRequests are the ones the user has sent?

Reply:
yes!
When you send a freind request, that other user ID gets pushed into your `pendingFriendRequest`
And from the other user POV, your ID gets pushed into their `friendRequests`

Reply:
Can you try to change them to `sentFriendRequests` and `receivedFriendRequests` ?